frontend/src/Explorer/View/Transaction.purs

```diff
@@ -70,7 +70,9 @@ summaryItems :: CTxSummary -> Language -> SummaryItems
 summaryItems (ctxSum@CTxSummary txSummary) lang =
     [ { label: translate (I18nL.tx <<< I18nL.txTime) lang
       , value: let  dateFormat = translate (I18nL.common <<< I18nL.cDateFormat) lang
-                    dateValue = fromMaybe noData <<< prettyDate dateFormat $ txSummary ^. ctsTxTimeIssued
+                    mDate      = txSummary ^. ctsTxTimeIssued
```

Comment:
> Seems you like composition a lot? Is that it?

Sure, it's one of the core features of FP. And it make sense, but not at all. I think it's good to go a middle way: Use it, but don't overuse it at cost of readable. 

The same for local variables: Use it if it really make sense (e.g. breaking a *long* chain of composition into more understandable parts). But avoid to use it as a `matter of principle`, such as storing simple functions into variables, if you can read / understand composition of functions well (e.g. `memPoolTxsSize` v.s. `HM.size memPoolTxs` - sorry, I have no better example :) ) 

As I sad before: It's all about a personal point of view :) And at the end: We can just learn from each others ;)